auth/oauth2_test.go

```diff
@@ -0,0 +1,52 @@
+package auth
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
There are no good/bad tests for EncodeHeader()?

Reply:
This requires contacting an oauth2 service a can't really be unit-tested. Stubbing this would be awkard, requiring changes to the API that will make it more difficult for normal users, and given that the method doesn't do much outside of communtion with external service, it's bascially not worth it here.